Flow.Launcher/SettingWindow.xaml.cs

```diff
@@ -297,5 +311,60 @@ private void OnExternalPluginInstallClick(object sender, RoutedEventArgs e)
                 textBox.MoveFocus(tRequest);
             }
         }
+
+        private void DarkModeSelectedIndexChanged(object sender, EventArgs e)
+        {
+            if (settings.DarkMode == "Light")
+            {
+                ModernWpf.ThemeManager.Current.ApplicationTheme = ModernWpf.ApplicationTheme.Light;
+            }
+            else if (settings.DarkMode == "Dark")
+            {
+                ModernWpf.ThemeManager.Current.ApplicationTheme = ModernWpf.ApplicationTheme.Dark;
+            }
+        }
+
+        /* Custom TitleBar */
+
+        private void OnMinimizeButtonClick(object sender, RoutedEventArgs e)
+        {
+            this.WindowState = WindowState.Minimized;
+        }
```

Comment:
why we need to add this😶‍🌫️

Reply:
because titlebar cannot change the color. So I had to remove default titlebar and made custom titlebar. In this case, there is no button,(maximize, close)  so it must be made. Other pop-up windows were applied like this.